packages/onboard-react/src/components/WalletDialog/WalletDialog.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FC, useCallback, useMemo, useState } from "react";
+import {
+  Dialog,
+  VerticalTabs,
+  VerticalTabsItem,
+  Button,
+  ButtonStyle,
+  Heading,
+  DialogSize,
+} from "@sovryn/ui";
+import { FilterType, WalletList } from "../WalletList/WalletList";
+import { connectWallet$ } from "@sovryn/onboard-core/dist/streams";
+import { InstructionsTab } from "../InstructionsTab/InstructionsTab";
+import styles from "./WalletDialog.module.css";
+
+type WalletDialogProps = {
+  isOpen: boolean;
+};
+
+const WalletDialog: FC<WalletDialogProps> = ({ isOpen }) => {
+  const [index, setIndex] = useState(0);
+
+  const items: VerticalTabsItem[] = useMemo(
+    () => [
+      {
+        label: "Hardware Wallet",
+        infoText: "Select the hardware wallet you want to connect",
+        content: <WalletList filter={FilterType.hardware} />,
+        dataLayoutId: "hardware",
+      },
+      {
+        label: "Browser Wallet",
+        infoText: "Select the web3 wallet you want to connect",
+        content: <WalletList filter={FilterType.browser} />,
+        dataLayoutId: "browser",
+      },
+      {
+        label: "Don't have a wallet?",
+        infoText: "Read the following instructions",
+        content: <InstructionsTab />,
+        dataLayoutId: "instructions",
+      },
+    ],
+    []
+  );
+
+  const handleCloseClick = useCallback(() => {
+    connectWallet$.next({
+      inProgress: false,
+    });
+  }, []);
+
+  return (
+    <Dialog
+      isOpen={isOpen}
+      onClose={handleCloseClick}
+      closeOnEscape
+      width={DialogSize.xl2}
+    >
+      <VerticalTabs
```

Comment:
VerticalTabItem is using vanilla `<p>` tag, please change it to use Paragraph/Heading component as appropriate instead.

All text in the dialog isn't using Roboto

Reply:
VerticalTabs are not part of the PR and repo, I think it should be reported as bug then :D

Dialog will be using parent body font which is roboto (as other components should unless it requires custom font), so there wouldn't be a need to edit all components if main font changes

Reply:
Agreed, will create another ticket for updating the VerticalTabs component with correct component usage - I'll update spec to indicate QA shouldn't check this part (we can have it tested as part of the full flow testing)